springcloud/src/test/java/org/axonframework/extensions/springcloud/commandhandling/SpringCloudCommandRouterTest.java

```diff
@@ -543,13 +559,15 @@ private void initAll(List<SpringCloudCommandRouter> routers) {
     private SpringCloudCommandRouter createRouterFor(String host) {
         Registration localServiceInstance = mock(Registration.class);
         when(localServiceInstance.getUri()).thenReturn(URI.create("http://" + host));
-        return SpringCloudCommandRouter.builder()
+        SpringCloudCommandRouter router = SpringCloudCommandRouter.builder()
                                        .discoveryClient(discoveryClient)
                                        .localServiceInstance(localServiceInstance)
                                        .routingStrategy(routingStrategy)
                                        .serviceInstanceFilter(serviceInstance -> true)
                                        .consistentHashChangeListener(consistentHashChangeListener)
                                        .build();
+        //router.updateMembership(LOAD_FACTOR, COMMAND_NAME_FILTER);
```

Comment:
I don't think this change here is necessary to be honest :-)

Reply:
Oops.  Leftover from me messing around.